lighthouse-core/audits/final-screenshot.js

```diff
@@ -30,18 +31,23 @@ class FinalScreenshot extends Audit {
    */
   static async audit(artifacts, context) {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
+    const traceOfTab = await TraceOfTab.request(trace, context);
     const screenshots = await Screenshots.request(trace, context);
     const finalScreenshot = screenshots[screenshots.length - 1];
 
     if (!finalScreenshot) {
       throw new LHError(LHError.errors.NO_SCREENSHOTS);
     }
 
+    // trace-of-tab uses microseconds for the timestamp, so we'll convert for consistency
+    const finalScreenshotTs = finalScreenshot.timestamp * 1000;
```

Comment:
`computed/screenshots.js` is [dividing the timestamp by 1000](https://github.com/GoogleChrome/lighthouse/blob/81a8a5cb3d5e785494fe8549cbe747de2325c17c/lighthouse-core/computed/screenshots.js#L22), and this is the only audit using that computed artifact (did we use it for something else at some point?), so, want to just not divide it there?

Reply:
yeah sg, I wanted to kill it at some point for being mostly useless, but that'll have to wait for another day :)